packages/commonwealth/client/scripts/views/components/Select/index.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../../../styles/shared';
+
+.select {
```

Comment:
usually we have main class capitalized at the top and then other classes are nested

```
.Select {
  .select-options-wrapper {...}
}
```

select-option should be probably in separate file

Reply:
`.select` and `.select-options-wrapper` are not nested in markup/html. `.select-options-wrapper` is portal'ed outside of root. Should we still make the classes nested?

Reply:
got ya - so no, in this case should not be nested